presto-main/src/test/java/io/prestosql/connector/MockConnectorFactory.java

```diff
@@ -217,7 +217,11 @@ public ConnectorTableLayout getTableLayout(ConnectorSession session, ConnectorTa
             @Override
             public Optional<ConnectorViewDefinition> getView(ConnectorSession session, SchemaTableName viewName)
             {
-                return Optional.of(getViews.apply(session, viewName.toSchemaTablePrefix()).get(viewName));
```

Comment:
Wouldn't be enough to leave the code as it was and just use `Optional.ofNullable`?